specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json

```diff
@@ -839,6 +839,9 @@
         "x-ms-examples": {
           "Enable Protection on Azure IaasVm": {
             "$ref": "./examples/AzureIaasVm/ConfigureProtection.json"
+          },
```

Comment:
200/201 doesn't come from this API. There are separate APIs for tracking and URLs for them is returned in location headers. 
It doesn't make sense to add 200/201 response here as this particular API will never return 200/201.

Reply:
The aforementioned api to retrieve the status is - https://github.com/Azure/azure-rest-api-specs/blob/39cf7c2508f56f0931c081ab25ee707678e04576/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json#L975

Reply:
@DheerendraRathor jfyi - it looks like the createOrUpdate that Vlad point is a long running operation (LRO). If we 
(1) Add 200 response model as ProtectedItemResource and 
(2) Mark this operation as long running (x-ms-long-running) 

then all language codegen can emit a method - upon its invocation it can initiate a request to create/update the resource and polls until the operation completes (using location header) then returns the final result to the caller. 

This is how ARM services in general expose LRO operation through SDK. You might want to consider this for consistency. I'll create an issue so that we can consider this in the next iteration.

Reply:
https://github.com/Azure/azure-rest-api-specs/issues/4032